Description: Correct compilation with gcc-6
 The patch assumes that a newline was the intended output after the 
 according message. 
Author: Gert Wollny <gw.fossdev@gmail.com> 
Forwarded:no
Debian-Bug: https://bugs.debian.org/811818

--- a/src/mod_video/crrc_animation.cpp
+++ b/src/mod_video/crrc_animation.cpp
@@ -84,7 +84,7 @@
       else
       {
         std::cerr << "createAnimation: unknown animation type '"
-                  << type << "'" << std::cerr;
+                  << type << "'" << std::endl;
       }
       
       if (anim != NULL)
